invenio_oauth2server/theme/semantic/form_styling.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of Invenio.
+# Copyright (C) 2015-2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Render semantic ui styling for forms."""
+
+from wtforms.widgets import HTMLString
+
+
+def semantic_style_dropdown(field, **kwargs):
+    """Render dropdown widget."""
+    html = [u'<div class="ui fluid dropdown button icon">']
+    html.append(u'<i class="dropdown icon"></i>')
+    html.append(u'<input type="hidden" name="client_type">')
+
+    for value, label, checked in field.iter_choices():
+        if checked:
+            html.append(
+                u'<div class="default text">{0}</div><div class="menu">'.format(
+                    label
+                )
+            )
+        html.append(
+            u'<div class="item" data-value="{0}">{0}</div>'.format(label)
```

Comment:
You probably need:
```suggestion
            u'<div class="item" data-value="{0}">{1}</div>'.format(value, label)
```
`